From: Roland Dreier <roland@purestorage.com>
Date: Mon, 15 Dec 2014 18:17:17 -0800
Subject: [PATCH 034/117] mlx5_core: Re-add MLX5_DEV_CAP_FLAG_ON_DMND_PG flag
Patch-mainline: v3.19-rc1
Git-commit: 6cb7ff3dcfe6aad6a36a0fd0e928b5bea4fabdd5
References: bsc#923036 FATE#318772

In commit 0c7aac854f52 ("net/mlx5_core: Remove unused dev cap enum
fields"), the flag MLX5_DEV_CAP_FLAG_ON_DMND_PG was removed.
Unfortunately the on-demand paging changes actually use it, so re-add
the missing flag.

Signed-off-by: Roland Dreier <roland@purestorage.com>
Signed-off-by: Hadar Hen Zion <hadarh@mellanox.com>
Acked-by: Cho, Yu-Chen <acho@suse.com>
---
 include/linux/mlx5/device.h |    1 +
 1 file changed, 1 insertion(+)

--- a/include/linux/mlx5/device.h
+++ b/include/linux/mlx5/device.h
@@ -225,6 +225,7 @@ enum {
 	MLX5_DEV_CAP_FLAG_APM		= 1LL << 17,
 	MLX5_DEV_CAP_FLAG_ATOMIC	= 1LL << 18,
 	MLX5_DEV_CAP_FLAG_BLOCK_MCAST	= 1LL << 23,
+	MLX5_DEV_CAP_FLAG_ON_DMND_PG	= 1LL << 24,
 	MLX5_DEV_CAP_FLAG_CQ_MODER	= 1LL << 29,
 	MLX5_DEV_CAP_FLAG_RESIZE_CQ	= 1LL << 30,
 	MLX5_DEV_CAP_FLAG_DCT		= 1LL << 37,
